### ISSUE TITLE
Create Notebook for **what are GANs, and what actually is a neural network?**

### ISSUE THREAD

**Issue by celiacintas** [NickleDave/generative-neural-nets-tutorial-scipy2020#12]
Here I'm gonna kickstart first iteration of notebook for neural nets with numpy and scipy following:
+ **what are GANs, and what actually is a neural network?** 45m
  - *instruction*, 15m: what is a GAN
    + *hands-on* example: load trained generator network, producing samples; explain: this is our goal!
    + instruction, 10 m: intuition behind generative adversarial training
      - adversarial game between generator and discriminator
        + role of the generator: generate samples that fool discriminator
        + input is vectors from latent space
        + role of discriminator: classify samples as real (data) or fake (output from generator)
      - build intuition with figures, concept maps
  - *instruction*, 30m: crash course in neural networks
    + along with live demo in Numpy
        - an artificial neuron with inputs, weights, bias, activation and output
        - a fully connected layer of artificial neurons
        - a multi-layer perceptron with input, hidden, and output layers
        - simplest form of fitting with Stochastic Gradient Descent
    + *hands-on* exercises, 5m: writing and training a perceptron with Numpy

**Comment by celiacintas** [comment]
I added the MNIST file here .. we should talk about which policy we should use for data?? all here at the repo with git-lfs? what do you think @NickleDave @pablo1n7 ??
